static/app/utils/integrationUtil.tsx

```diff
@@ -207,7 +209,20 @@ export const getIntegrationIcon = (integrationType?: string, size?: string) => {
 
 // used for project creation and onboarding
 // determines what integration maps to what project platform
-export const platfromToIntegrationMap = {
+export const platformToIntegrationMap = {
   'node-awslambda': 'aws_lambda',
   'python-awslambda': 'aws_lambda',
 };
+
+export const isSlackIntegrationUpToDate = (integrations: Integration[]): boolean => {
+  return integrations.every(
+    integration =>
+      integration.provider.key !== 'slack' || integration.scopes?.includes('commands')
+  );
+};
```

Comment:
nit: If in the future we add or remove some more permissions from the Slack integration, this would need to be updated to include more conditionals. 

Should we just check all the scopes against all the required scopes on the backend and ensure they match 1 for 1? Probably overboard, but just a thought.